docs/tutorial/data.md

```diff
@@ -320,6 +320,60 @@ data.add_df(df)
     You'll need to adjust the SQL query and the database connection parameters
     to match your specific use case.
 
+### Using `pyspark` DataFrame
+
+Use
+[`add_spark_df`](../reference/ipyvizzu/animation.md#ipyvizzu.animation.Data.add_spark_df)
+method for adding `pyspark` DataFrame to
+[`Data`](../reference/ipyvizzu/animation.md#ipyvizzu.animation.Data).
+
+```python
+from pyspark.sql import SparkSession
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    StringType,
+    IntegerType,
+)
+from ipyvizzu import Data
+
+
+spark = SparkSession.builder.appName("ipyvizzu").getOrCreate()
+spark_schema = StructType(
+    [
+        StructField("Genres", StringType(), True),
+        StructField("Kinds", StringType(), True),
+        StructField("Popularity", IntegerType(), True),
+    ]
+)
+spark_data = [
+    ("Pop", "Hard", 114),
+    ("Rock", "Hard", 96),
+    ("Jazz", "Hard", 78),
+    ("Metal", "Hard", 52),
+    ("Pop", "Smooth", 56),
+    ("Rock", "Experimental", 36),
+    ("Jazz", "Smooth", 174),
+    ("Metal", "Smooth", 121),
+    ("Pop", "Experimental", 127),
+    ("Rock", "Experimental", 83),
+    ("Jazz", "Experimental", 94),
+    ("Metal", "Experimental", 58),
+]
+df = spark.createDataFrame(spark_data, spark_schema)
+
+data = Data()
+data.add_spark_df(df)
```

Comment:
Why is not used the same `add_df` function for spark dataframe too? The interface can be made simpler.

Reply:
@thanks your suggestion, implemented in with Protocol https://github.com/vizzuhq/ipyvizzu/commit/8926117d81be355e8910482fa348bd1e6f7915ba

Reply:
@schaumb thanks the suggestion, implemented in https://github.com/vizzuhq/ipyvizzu/commit/8926117d81be355e8910482fa348bd1e6f7915ba